<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Clients">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Magasins</string>
     </attribute>
     <widget class="QTabWidget" name="Tabs_Mag">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>701</width>
        <height>431</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Ajout_Mag">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="Ajout_Mag_btn">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>360</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_12">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>81</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_4">
         <item>
          <widget class="QLineEdit" name="ID_Mag_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Nom_Mag_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Location_Mag_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_11">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>71</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_3">
         <item>
          <widget class="QLabel" name="ID_Magasin_2">
           <property name="text">
            <string>ID :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Nom_Magasin_2">
           <property name="text">
            <string>Nom :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie_2">
           <property name="text">
            <string>Categorie :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Location_2">
           <property name="text">
            <string>Location :</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Modif_Mag">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>81</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_2">
         <item>
          <widget class="QLineEdit" name="ID_Mag"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Nom_Mag"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Location_Mag"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>71</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs">
         <item>
          <widget class="QLabel" name="ID_Magasin">
           <property name="text">
            <string>ID :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Nom_Magasin">
           <property name="text">
            <string>Nom :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie">
           <property name="text">
            <string>Categorie :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Location">
           <property name="text">
            <string>Location :</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="Modif_Mag_btn">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>360</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QPushButton" name="Supp_Mag_btn">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_13">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>140</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_5">
         <item>
          <widget class="QLabel" name="ID_Magasin_3">
           <property name="text">
            <string>ID :</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_14">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>130</y>
          <width>81</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_6">
         <item>
          <widget class="QLineEdit" name="Location_Mag_3"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>631</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf_btn">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>340</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Enregistrer en pdf</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tri</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Recherche</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Statistique</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Magasin">
     <attribute name="title">
      <string>Clients</string>
     </attribute>
     <widget class="QTabWidget" name="Tabs_Clts">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>701</width>
        <height>431</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Ajout_Mag_5">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="on_Ajout_Mag_btn_5_clicked">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>360</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_15">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>71</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_31">
         <item>
          <widget class="QLabel" name="ID_Magasin_16">
           <property name="text">
            <string>ID :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Nom_Magasin_11">
           <property name="text">
            <string>Nom :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie_14">
           <property name="text">
            <string>Prenom :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie_15">
           <property name="text">
            <string>Sexe :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie_16">
           <property name="text">
            <string>Email :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie_17">
           <property name="text">
            <string>Num.Tel :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Location_11">
           <property name="text">
            <string>Pts.Fid :</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_16">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>81</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_32">
         <item>
          <widget class="QLineEdit" name="ID_Mag_11"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Nom_Mag_11"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag_14"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag_15"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag_16"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag_17"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Location_Mag_16"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Modif_Mag_5">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_9">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>81</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_27">
         <item>
          <widget class="QLineEdit" name="ID_Mag_10"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Nom_Mag_10"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag_10"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag_11"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag_12"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Categorie_Mag_13"/>
         </item>
         <item>
          <widget class="QLineEdit" name="Location_Mag_14"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_10">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>71</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_28">
         <item>
          <widget class="QLabel" name="ID_Magasin_14">
           <property name="text">
            <string>ID :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Nom_Magasin_10">
           <property name="text">
            <string>Nom :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie_10">
           <property name="text">
            <string>Prenom :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie_12">
           <property name="text">
            <string>Sexe :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie_13">
           <property name="text">
            <string>Email :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Categorie_11">
           <property name="text">
            <string>Num.Tel :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Location_10">
           <property name="text">
            <string>Pts.Fid :</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="Modif_Mag_btn_5">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>360</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_21">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QPushButton" name="Supp_Mag_btn_5">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_29">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>140</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_29">
         <item>
          <widget class="QLabel" name="ID_Magasin_15">
           <property name="text">
            <string>ID :</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_30">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>130</y>
          <width>81</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Magasin_inputs_30">
         <item>
          <widget class="QLineEdit" name="Location_Mag_15"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_22">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>631</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf_btn_5">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>340</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Enregistrer en pdf</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_23">
       <attribute name="title">
        <string>Tri</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_24">
       <attribute name="title">
        <string>Recherche</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_25">
       <attribute name="title">
        <string>Statistique</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCentre_Commercial">
    <property name="title">
     <string>Centre Commercial</string>
    </property>
   </widget>
   <addaction name="menuCentre_Commercial"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
